utils/lit/tests/Inputs/shtest-format/cond_run_requires-missing.txt

```diff
@@ -0,0 +1,2 @@
+COND_RUN(a-missing-feature): true
+COND_RUN(a-missing-feature, a-present-feature): true
```

Comment:
Does `true` simply return a non-failing exit code?  If so, don't we want a failing exit code to ensure it was skipped?